src/common/Configuration/Config.cpp

```diff
@@ -469,25 +469,25 @@ bool ConfigMgr::LoadModulesConfigs(bool isReload /*= false*/, bool isNeedPrintIn
     return true;
 }
 
-// @deprecated DO NOT USE - use GetOption<std::string> instead.
+/// @deprecated DO NOT USE - use GetOption<std::string> instead.
```

Comment:
May I ask why three?

Reply:
To trigger doxygen

Reply:
Doxygen triggers on these types of comments,
```
/** 
*
*
*/
```

and

```
///
```

`@deprecated` will contribute to https://www.azerothcore.org/pages/doxygen/d8/d38/a04419.html
`@bug` to https://www.azerothcore.org/pages/doxygen/d9/d51/a04420.html
`@todo` to https://www.azerothcore.org/pages/doxygen/d3/d74/a04418.html